Sources/PrimerSDK/Classes/Core/Analytics/AnalyticsService.swift

```diff
@@ -141,14 +146,14 @@ extension Analytics {
 
                             seal.fulfill()
 
-                            if remainingEvents.count >= self.batchSize {
-                                self.sync(events: remainingEvents)
+                            if events.count >= self.batchSize {
```

Comment:
This should be reverted to `remainingEvents`

Reply:
Resolved. Thanks!